web/forms/contact_form.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Community.ContactForm do
+  use Community.Web, :model
+
+  embedded_schema do
+    field :body, :string
+    field :email, :string
+    field :subject, :string
+    field :member
+  end
+
+  @required_fields [
+    :body,
+    :email,
+    :member,
+    :subject,
+  ]
```

Comment:
If you want you can use `~w()a` note the `a` at the end. That is a sigil modifier that tells it to creates atoms instead of strings